OAuth docs tweaks (#2679)
bluesky-social/atproto: packages/oauth/oauth-client-node/README.md

# atproto OAuth Client for NodeJS

This package implements all the OAuth features required by [ATPROTO] (PKCE,

> etc.) to run in a NodeJS based environment. This could include desktop apps
> built with Electron, or traditional web app backends built with frameworks like
> Express.

Reviewer: ```suggestion
etc.) to run in a NodeJS based environment such as desktop apps built with
Electron or traditional web app backends built with frameworks like Express.
```
